public/app/percona/backup/components/AddBackupPage/AddBackupPage.styles.ts

```diff
@@ -0,0 +1,75 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getStyles = ({ colors, shape, spacing, v1: { spacing: spacingV1 } }: GrafanaTheme2) => ({
+  pageWrapper: css`
+    max-width: 1440px;
```

Comment:
I realised you're not using `OldPage`, in the page component, like in other places. Can you try to add it and check if this is still needed?

Reply:
I can't use OldPage component like in other places because this page looks different than other places. I copied the same style as it is in AlertRuleForm because it is the same logic with a new page instead of a modal. But to make it nicer I changed it from hardcoded value to ${breakpoints.values.xxl} I did not notice this the first time.

Reply:
Oh, right, 'cause it's a different page, you're right